### ISSUE TITLE
Keyboard focus is lost when using CMD-1/2/3

### ISSUE THREAD

**Issue by psculthorpe** [Ranchero-Software/NetNewsWire#1009]
If I use CMD-1/2/3 to switch between Today/All unread/Starred, then the arrows on the keyboard no longer behave correctly. I have to press Tab at least twice for the side bar highlight to reappear (depending on if I have the toolbar visible or not). I cannot see where keyboard focus has moved to when pressing the shortcut.

Should the focus be whichever option I have just switched to rather than hidden/lost?

**Comment by brentsimmons** [comment]
Which version of macOS are you running?

**Comment by Wevah** [comment]
It looks like if the article list has focus when you press Cmd-1/2/3, focus jumps to the detail view instead of remaining on the article view.

**Comment by vincode-io** [comment]
This is in the upcoming 5.1. release.
